src/context_store/ingestion/pipeline.py

```
@@ -0,0 +1,271 @@
+
+        # ステップ7: グラフノード作成
+        await self._graph.create_node(
+            memory_id,
+            {
+                "memory_type": memory.memory_type.value,
+                "source_type": memory.source_type.value,
+                "project": memory.project,
+            },
+        )
+
+        # ステップ8: グラフリンク（エッジ作成）
```

Ruff (0.15.7)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)